File: Backend/BusinessLogic/Models/SessionSeatModel.cs
using System;

namespace BusinessLogic.Models
{
    public class SessionSeatModel
    {
        public int SeatId { get; set; }
        public int SessionId { get; set; }
        public int UserId { get; set; }
        public SessionSeatStatus Status { get; set; }
        public DateTime TakenAt { get; set; }
        public int Row { get; set; }
        public int Place { get; set; }
        public int SeatTypeId { get; set; }
    }
}

File: Frontend/src/app/Models/SessionSeatModel.ts
import { SessionSeatStatuses } from "./SessionSeatStatuses";

export class SessionSeatModel {
    userId : number = 0;
    seatId : number = 0;
    sessionId : number = 0;
    row : number = 0;
    place : number = 0;
    seatTypeId : number = 0;
    status : SessionSeatStatuses = SessionSeatStatuses.Free;
}